src/main/scala/net/cakesolutions/CakeStandardsPlugin.scala

```diff
@@ -71,9 +70,8 @@ object CakeStandardsPlugin extends AutoPlugin {
       Warts.unsafe ++ Seq(
         Wart.FinalCaseClass,
         Wart.ExplicitImplicitTypes
-      )
+      ),
+    // https://github.com/HairyFotr/linter#enablingdisabling-checks
```

Comment:
it was conflicting with wartremover... projects can opt-in if they want but we had troubles.